olive/engine/engine.py

```diff
@@ -254,6 +254,7 @@ def register_pass(
     def run(
         self,
         input_model: OliveModel,
+        data_root: str = None,
```

Comment:
Double check: data_root accepts only `str` and all the operation about resource type will be after `normalize_data_path`, right?

Reply:
yes